.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -10,26 +10,29 @@ MUST: Replace *everything* in angle brackets in the title AND body of this issue
 
 If you have any questions see the [cherry-pick documentation](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#cherry-picks).
 -->
+
 # Cherry-pick request
 
 <!--
 TIP: Cherry-picks into production most likely require a cherry-pick into RC too. Otherwise, your fix will be lost when the RC is promoted.
 -->
 
-| Issue | PR  | Production? | RC? | [Release issue](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) |
-| :---: | :-: | :---------: | :-: | :-----------: |
-| #<_ISSUE_NUMBER_> | #<_PR_NUMBER_> | **<YES/NO>** | **<YES/NO>** | #<_RELEASE_ISSUE_> |
+|       Issue       |       PR       | Production?  |     RC?      | [Release issue](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) |
```

Comment:
What's going on with the spacing here?  I assumed it was trying to give the same space to each column, but the hyphens below don't look the same length.